distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Filter_Condition_Helpers.enso

```diff
@@ -67,5 +84,20 @@ make_filter_column source_column filter_condition on_problems = case filter_cond
         if is_nan_column.is_error then is_infinite_column.not else
             (is_infinite_column || is_nan_column).not
     # Vector
-    Is_In values -> source_column.is_in values
-    Not_In values -> source_column.is_in values . not
+    Is_In values ->
+        warn_on_nothing_in_comparison filter_condition values <|
+            source_column.is_in values
+    Not_In values ->
+        warn_on_nothing_in_comparison filter_condition values <|
+            source_column.is_in values . not
+
+warn_on_nothing_in_comparison : Filter_Condition -> Any | Vector Any -> Any -> Any
+warn_on_nothing_in_comparison filter_condition x ~action =
+    result = action
+    has_nothing = case x of
+        _ : Column -> False
+        _ : Database_Column -> False
+        _ : Vector -> x.contains Nothing
```

Comment:
Shouldn't this case only matter in `Is_In` and not the others??

If I do
```
table.filter "my_vectors" (Filter_Condition.Equal [Nothing, Nothing])
```
this uses the generic equality and should not warn, right?

Reply:
Good catch -- done.